### PR TITLE
improved login ui

### PR DESCRIPTION
**Pull request by at-lytruong**
improved login, register, forgot ui
add image logo and background

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/View/Controllers/Login/LoginViewController.swift [hunk 1]
```diff
@@ -19,11 +19,17 @@ class LoginViewController: UIViewController {
     }
 
     @IBAction func signInTouchUpInSide(_ sender: Any) {
+        let tabbar = TabBarViewController()
+        self.present(tabbar, animated: true, completion: nil)
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/LoginViewController.swift]: remove self

File: MyApp/View/Controllers/Login/LoginViewController.swift [hunk 2]
```diff
@@ -19,11 +19,17 @@ class LoginViewController: UIViewController {
...
     }
 
     @IBAction func forgotTouchUpInSide(_ sender: Any) {
+        let forgot = ForgotPassWordViewController()
+        self.present(forgot, animated: true, completion: nil)
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/LoginViewController.swift]: self

File: MyApp/View/Controllers/Login/LoginViewController.swift [hunk 3]
```diff
@@ -19,11 +19,17 @@ class LoginViewController: UIViewController {
...
     }
 
     @IBAction func signUpTouchUpInSide(_ sender: Any) {
+        let signUp = RegisterViewController()
+        self.present(signUp, animated: true, completion: nil)
```
Comment (at-hieutran) [inline MyApp/View/Controllers/Login/LoginViewController.swift]: self

**Comment by at-hieutran** [comment]
LGTM

**Pull request merged**